tests/sc/test_report_definition.py

```diff
@@ -0,0 +1,30 @@
+'''
+test file to test various scenarios in sc report_definition
+'''
+import pytest
+from ..checker import check
+
+def test_report_definition_id_typeerror(security_center):
+    '''
+    validate id is int not string
+    '''
+    with pytest.raises(TypeError):
+        security_center.report_definition.launch(id='one')
+
+def test_report_definition_id_item_typeerror(security_center):
+    '''
+    validate id is int not list
+    '''
+    
+    with pytest.raises(TypeError):
+        security_center.report_definition.launch(id=['one'])
+
+def test_report_definition_launch_success(security_center):
```

Comment:
Solves https://github.com/tenable/pyTenable/actions/runs/11962140227/job/33349851654?pr=656#step:7:808

```suggestion

@pytest.mark.vcr()
def test_report_definition_launch_success(security_center):
```